assets/js/base/components/cart-checkout/product-name/style.scss

```diff
@@ -2,5 +2,5 @@
 	@include font-size(16);
 	@include wrapBreakWord();
 	display: block;
-	width: max-content;
+	max-width: max-content;
```

Comment:
not defining a width could also be an alternative

Reply:
Yeah... for reference: that declaration makes the link clickable area smaller so there are no accidental clicks.